dev/io.openliberty.rest.handler.config.openapi.2.0/src/io/openliberty/rest/handler/config/openapi20/ConfigSchemaRESTHandler.java

```diff
@@ -0,0 +1,103 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.rest.handler.config.openapi20;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.eclipse.microprofile.openapi.models.OpenAPI;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.wsspi.rest.handler.RESTHandler;
+import com.ibm.wsspi.rest.handler.RESTRequest;
+import com.ibm.wsspi.rest.handler.RESTResponse;
+
+import io.smallrye.openapi.runtime.OpenApiProcessor;
+import io.smallrye.openapi.runtime.OpenApiStaticFile;
+import io.smallrye.openapi.runtime.io.Format;
+import io.smallrye.openapi.runtime.io.OpenApiSerializer;
```

Comment:
I don't have a better idea right now but I don't like us continuing to leak openapi internals into other components.

Reply:
We could do, but we would be creating a new API to use internally just for this.

Similarly, there is quite a bit of shared code across mpOpenAPI 1.x and 2.0, they could have introduced an API there too.